src/aspire/volume/volume.py

```diff
@@ -146,6 +153,35 @@ def __setitem__(self, key, value):
         self._check_key_dims(key)
         self._data[key] = value
 
+    @property
+    def symmetry_group(self):
+        """
+        A SymmetryGroup instance associated with the symmetry of the volume.
+        Access rotation matrices of the symmetry_group via `symmetry_group.rotations`.
+        """
+        return self._symmetry_group
+
+    @symmetry_group.setter
```

Comment:
Good question. This doesn't actually allow setting after volume creation because it raises an error if a Volume already has a `symmetry_group` attribute (which would be the case for an instantiated Volume), but I guess that defeats the purpose of making this a setter. I've changed this to a private method `_set_symmetry_group` that checks some logic and sets the `symmetry_group` attribute during Volume init.

Reply:
Ok makes sense.